cmd/helm/uninstall.go

```diff
@@ -72,6 +72,7 @@ func newUninstallCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&client.DisableHooks, "no-hooks", false, "prevent hooks from running during uninstallation")
 	f.BoolVar(&client.KeepHistory, "keep-history", false, "remove all associated resources and mark the release as deleted, but retain the release history")
 	f.BoolVar(&client.Wait, "wait", false, "if set, will wait until all the resources are deleted before returning. It will wait for as long as --timeout")
+	f.BoolVar(&client.Cascade, "cascade", false, "if set, all dependent resources are deleted before the deletion is completed")
```

Comment:
Should we mimic `kubectl delete --cascade`:


```
    --cascade='background':
        Must be "background", "orphan", or "foreground". Selects the deletion cascading strategy for the dependents
        (e.g. Pods created by a ReplicationController). Defaults to background.
 ```

Reply:
Thanks @gjenkins8 for your suggestion. I have updated the implementation accordingly